src/Truncate/index.jsx

```diff
@@ -66,4 +66,9 @@ Truncate.defaultProps = {
   onTruncate: undefined,
 };
 
+function Truncate() {
+  return null;
```

Comment:
[consideration] I wonder if it'd be worth having something like a `console.log` or similar here to give some output when this no-op component is used, e.g.

```jsx
function Truncate() {
  useEffect(() => {
    console.log('Please use Truncate.Deprecated until a replacement is created');
  }, []);
  return null;
}
```